### ISSUE TITLE
Compilation error (No instance for (Applicative (DocM s)))

### ISSUE THREAD

**Issue by tchoutri** [amtal/CoreErlang#4]
Hi! I would like to use your library to make a type analyzer for BEAM-based languages (Erlang, Elixir, LFE…) that would use Core Erlang representations of the code.
I started a `stack` project, but when I try to build it (wihtout touching anything), I get:

```
Configuring CoreErlang-0.0.2...
Building CoreErlang-0.0.2...
Preprocessing library CoreErlang-0.0.2...
[1 of 3] Compiling Language.CoreErlang.Syntax ( Language/CoreErlang/Syntax.hs, .stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build/Language/CoreErlang/Syntax.o )
[2 of 3] Compiling Language.CoreErlang.Pretty ( Language/CoreErlang/Pretty.hs, .stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build/Language/CoreErlang/Pretty.o )

Language/CoreErlang/Pretty.hs:78:10: error:
    • No instance for (Applicative (DocM s))
        arising from the superclasses of an instance declaration
    • In the instance declaration for ‘Monad (DocM s)’

```

The weird thing is that it didn't happen when I compiled the library in its git repo. (  ⋅ _ .)

**Comment by amtal** [comment]
Hey! That's been resolved in 664a4750b733149053e329652f539bc95a143ac5 and pushed to hackage as CoreErlang-0.0.3.

**Comment by tchoutri** [comment]
Thank you so much! Have a nice day!
